smoke-test/tests/read_only/test_search.py

```diff
@@ -112,8 +119,31 @@ def test_search_works(entity_type, api_name):
         """,
         "variables": {"input": first_urn},
     }
+
     response = frontend_session.post(f"{get_frontend_url()}/api/v2/graphql", json=json)
     response.raise_for_status()
     res_data = response.json()
     assert res_data["data"], f"res_data was {res_data}"
     assert res_data["data"][api_name]["urn"] == first_urn, f"res_data was {res_data}"
+
+    print(f"Response entity_type {entity_type}")
```

Comment:
please make this a separate test for OpenAPI v3. So if the graphql tests fail this still runs

Reply:
Okay